### ISSUE TITLE
tile command broken in version 4.3.1

### ISSUE THREAD

**Issue by Kris-LIBIS** [minimagick/minimagick#336]
Somewhere between 4.2.10 and 4.3.1 a change broke my application. The imagemagick command generated when the tile method is called is different and no longer works correctly:

For this test:
```ruby
require 'mini_magick'

MiniMagick.configure do |config|
  config.debug = true
end

MiniMagick::Tool::Convert.new do |convert|
  convert.size("480x480")
  convert << 'xc:transparent'
  convert.tile('/tmp/wm.png')
  convert.draw "rectangle 0,0,480,480"
  convert.format(:JPG)
  convert << '/tmp/out.jpg'
end
```

With version <= 4.2.10 it generates:
```
convert -size 480x480 xc:transparent -tile /tmp/wm.png -draw rectangle 0,0,480,480 -format JPG /tmp/out.jpg
```

With version >= 4.3.1 it generates:
```
convert -size 480x480 xc:transparent tile:/tmp/wm.png -draw rectangle 0,0,480,480 -format JPG /tmp/out.jpg
```

I have searched through the commits, but I cannot find the change that caused this. I hope you have a clue on how to fix this.

**Comment by Kris-LIBIS** [comment]
Hmm. Looks like f444d2887563f8f130d5358f90a51b2da3cfa07f in file lib/mini_magick/tool.rb could be the culprit. Any thoughts?

**Comment by janko-m** [comment]
Thanks for a great issue report. I released 4.3.4 with the fix.
